projects/media-viewer/src/lib/annotations/models/tag-item.model.ts

```diff
@@ -0,0 +1,6 @@
+export interface TagItemModel {
+  name: string;
+  label: string;
+  color?: string;
+  di: string;
```

Comment:
should this be id or is it an abbreviation?

Reply:
removed